hasura.planx.uk/metadata/tables.yaml

```diff
@@ -248,6 +248,29 @@
           - user_context
           - user_data
       comment: Allow users who want to leave feedback on their experience to write to the table
+  select_permissions:
```

Comment:
nit/question: Are these select permissions needed for the mutation `DeleteFeedback` to work? It looks like we might need `id` for the `returning` clause but no more?

Reply:
Ahahahaha I also wasn't sure about this 😅 

I added the full column permissions when I was having issues. I couldn't seem to get a straight answer from the docs and I saw that other sanitised tables had the full select permission. 

I can work this out by restricting access and testing it not that the feature is working.

Reply:
Thanks again for picking up on this @DafyddLlyr I kind of wanted to do this testing but was worried I'd been taking too long on this piece of work. So it was a great excuse to try it out!

As discussed it seems the delete requires select permissions for any columns it references i.e. `created_at` for the `where` and `id` for the return value. 

I've updated the permissions to reflect this: b259adcb7545487aab16b61284acaf0a5e87ae5c